src/App/Pages/Accounts/RegisterPageViewModel.cs

```diff
@@ -128,8 +163,15 @@ public async Task SubmitAsync(bool showLoading = true)
                     AppResources.AnErrorHasOccurred, AppResources.Ok);
                 return;
             }
-
-            // TODO: Password strength check?
+            if (MasterPasswordStrength == PasswordStrengthCategory.Weak)
```

Comment:
Whew you have the reference to `PasswordStrengthViewModel` here you can directly use `PasswordStrengthViewModel. PasswordStrengthCategory == PasswordStrengthCategory.Weak`